qa/tasks/cephfs/test_fsnew.py

```diff
@@ -0,0 +1,64 @@
+import errno
+import time
+from unittest import SkipTest
+from tasks.cephfs.cephfs_test_case import CephFSTestCase
+from teuthology.orchestra.run import CommandFailedError
+
+class TestFSNew(CephFSTestCase):
+    def test_fs_new(self):
+        data_pool_name = "{0}_data".format(self.fs.name)
+
+        self.fs.mds_stop()
+        self.fs.mds_fail()
+
+        self.fs.mon_manager.raw_cluster_cmd('fs', 'rm', self.fs.name,
+                                            '--yes-i-really-mean-it')
+
+        self.fs.mon_manager.raw_cluster_cmd('osd', 'pool', 'delete',
+                                            self.fs.metadata_pool_name,
+                                            self.fs.metadata_pool_name,
+                                            '--yes-i-really-really-mean-it')
+        self.fs.mon_manager.raw_cluster_cmd('osd', 'pool', 'create',
+                                            self.fs.metadata_pool_name,
+                                            self.fs.get_pgs_per_fs_pool().__str__())
+
+        dummyfile = '/etc/fstab'
+
+        self.fs.put_metadata_object_raw("key", dummyfile)
```

Comment:
You could just use an inline string like "foobar" instead of dummyfile here -- put_metadata_object_raw isn't reading a file or anything like that.